### DISCUSSION TITLE
MRF Analytic Report - community feedback requested

### DISCUSSION THREAD

**Discussion by dgolden** [CMSgov/price-transparency-guide#630] [Show and tell]
Keyark.com is ready to release our first report, and we’d appreciate community feedback on the core reporting elements, as shown below.

The MRF Analytic Report presents in-network negotiated rates for common billing codes, delivered for provider group(s) of interest. The data is delivered in an Excel workbook, ready for use. The screenshot shows Aetna’s contract rates with a large provider group in Illinois, for an MRF updated on 2022-10-05.

![image](https://user-images.githubusercontent.com/6385204/199563473-20a5c412-8e0c-405b-9223-5a0be1ed82c4.png)

**Comment by hmgpa** [comment]
Connecting the Negotiated rate table with Zip Code and Provider ID either NPI or Tax ID would be useful.   Linking the NPI to actual provider ID would allow providers to see what their competitors have negotiated for the same services.  It would also allow plan administrators to assign a cost to patient rating.  Knowing which providers cost me less as a patient is the intent of the transparency project.
